cse110-fa21-group17/cse110-fa21-group17
# added signup logic and login frontend ejs files

# Administrative Info

## Changes
What changes did you make?
- Added GET `/signup` route that displays the signup page as a simple form
- Added POST `/signup` route that sends the user information to the backend and encrypt the password
- Added frontend pages for frontend team to style

## Dependencies/Library installed: 
- Bcrypt-nodejs

## How To Test:
1. make sure .env is up to date
2. run `npm install` under app/ subdirectory
3. run `nodemon` to start the server
4. Go to browser, visit: `localhost:3000/signup`, fill out the form and hit submit, it should direct you to the page saying signup successful. 

## Checklist

Remove any items which are not applicable.

- [x] I have added tested my program thoroughly
- [x] Did you add/sub fields in the environment file? if so please update .env_sample
